### PR TITLE
Review & update multi-node howto

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Review and update the Multinode How-To.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Partial: #60

### CODE REVIEW

**Comment by Loquacity** [comment]
image

**Comment by Loquacity** [comment]
@NunoFilipeSantos and @erimatnor Now would be a good time for you to jump in and propose some text for the bits I don't have yet 😉

**Comment by NunoFilipeSantos** [comment]
Thank you @Loquacity ! 🚀
@erimatnor can we slot this work is for this week?

**Comment by Loquacity** [comment]
@NunoFilipeSantos & @erimatnor bump?

**Review by NunoFilipeSantos — APPROVED** [review]
LGTM 👍

**Comment by Loquacity** [comment]
@NunoFilipeSantos Do you have more content for me for where the placeholders are?

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-grow-shrink.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Grow and shrink multinode
+Quickly re-engineer cutting-edge e-markets after low-risk high-yield
```
Comment (erimatnor) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-grow-shrink.md:2]: We have some things written on this topic here: https://docs.timescale.com/timescaledb/latest/how-to-guides/distributed-hypertables/manage-data-nodes/#moving-data-between-nodes

Should we move it here?

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-ha.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# High availability (HA) with multinode
+Efficiently myocardinate multidisciplinary meta-services and just in time action
```
Comment (erimatnor) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-ha.md:2]: We already have this section on "native replication": https://docs.timescale.com/timescaledb/latest/how-to-guides/distributed-hypertables/enable-native-replication/

But I think it makes sense to also talk more generally about HA here, including different options and alternatives.

**Review by erimatnor — COMMENTED** [review]
Made a quick pass to add some suggestions. I should be able to contribute more later.

**Comment by Loquacity** [comment]
Distributed hypertables is now part of hypertables (some content moved to hypertables or multi-node sections):
image

**Comment by Loquacity** [comment]
This can be merged as soon as everyone is ready for it to go live! 🎉

**Pull request merged**